kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,106 @@ This sample demonstrates how to install the Traefik ingress controller to provid
+### 1. Install WebLogic domains
+First, we need to prepare two domains for Traefik load balancing.
 
-Create two WLS domains:
-- One domain with name `domain1` under namespace `default`.
-- One domain with name `domain2` under namespace `test1`.
+Create two WebLogic domains:
+- One domain with `domain1` as the domain UID and namespace `weblogic-domain1`.
+- One domain with `domain2` as the domain UID and namespace `weblogic-domain2`.
 - Each domain has a web application installed with the URL context `testwebapp`.
+- Each domain has a WebLogic cluster `cluster-1` where each Managed Server listens on port `8001`.
 
-### 2. Install the Traefik Ingress
-#### Install a host-routing Ingress
+### 2. Web request routing 
+The following sections describe how to route an application web request to the WebLogic domain through a Traefik frontend.
+
+#### Host-based routing 
+This sample demonstrates how to access an application on two WebLogic domains using host-based routing. Install a host-based routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute)
 ```
 $ kubectl create -f samples/host-routing.yaml
+ingressroute.traefik.containo.us/traefik-hostrouting-1 created
+ingressroute.traefik.containo.us/traefik-hostrouting-2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different hostnames.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different hostnames as defined in route section of `host-routing.yaml`.
 ```
-$ curl -H 'host: domain1.org' http://${HOSTNAME}:30305/testwebapp/
-$ curl -H 'host: domain2.org' http://${HOSTNAME}:30305/testwebapp/
+# Get the ingress controller web port
+$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')
+$ curl -H 'host: domain1.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
+$ curl -H 'host: domain2.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
 ```
-#### Install a path-routing Ingress
+
+#### Path-based routing  
+This sample demonstrates how to access an application on two WebLogic domains using path-based routing. Install a path-based routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute) and [Middleware](https://docs.traefik.io/middlewares/overview/).
+
 ```
 $ kubectl create -f samples/path-routing.yaml
+ingressroute.traefik.containo.us/traefik-pathrouting-1 created
+middleware.traefik.containo.us/middleware-domain1 created
+ingressroute.traefik.containo.us/traefik-pathrouting-2 created
+middleware.traefik.containo.us/middleware-domain2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different paths.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different paths, as defined in the route section of the `path-routing.yaml`.
```

Comment:
the `path-routing.yaml`. -> the `path-routing.yaml` file.

Reply:
done